lib/plugins/alignedTable.js

```diff
@@ -14,7 +14,22 @@ function getLine(state, line) {
   var pos = state.bMarks[line] + state.blkIndent,
       max = state.eMarks[line];
 
-  return state.src.substr(pos, max - pos);
+  var indentation = getIndentation(state, line);
+  var lineText = state.src.substr(pos, max - pos);
+
+  if (indentation > 0) {
+    return removeIndentation(lineText, indentation);
+  }
+
+  return lineText;
+}
+
+function getIndentation(state, line) {
+  return state.tShift[line] || 0;
+}
+
+function removeIndentation(text, indentation) {
+  return (text || '').replace(new RegExp('\ {'+ indentation +'}', 'g'), '');
```

Comment:
I'd vote to pre-generate these RegExp(s) with various lengths of indentation (it's only a number, right?) and use a hash object as a cache for them.

Otherwise we're dealing with a not-so-small performance impact here.